src/styles/main.styles.scss

```diff
@@ -0,0 +1,8 @@
+* {
+  margin: 0;
```

Comment:
no need to create separate branch for styles if you store them with components
also honestly this styles are redudant